administration/src/bp-modules/cards/hooks/useCardGenerator.test.tsx

```diff
@@ -11,19 +11,44 @@ import createCards, { CreateCardsError, CreateCardsResult } from '../../../cards
 import deleteCards from '../../../cards/deleteCards'
 import { DynamicActivationCode, StaticVerificationCode } from '../../../generated/card_pb'
 import { Region } from '../../../generated/graphql'
+import { ProjectConfigProvider } from '../../../project-configs/ProjectConfigContext'
 import bayernConfig from '../../../project-configs/bayern/config'
+import { ProjectConfig } from '../../../project-configs/getProjectConfig'
+import koblenzConfig from '../../../project-configs/koblenz/config'
+import nuernbergConfig from '../../../project-configs/nuernberg/config'
+import showcaseConfig from '../../../project-configs/showcase/config'
 import downloadDataUri from '../../../util/downloadDataUri'
 import { AppToasterProvider } from '../../AppToaster'
 import useCardGenerator from './useCardGenerator'
 
-const wrapper = ({ children }: { children: ReactNode }) => (
-  <MemoryRouter>
-    <AppToasterProvider>
-      <ApolloProvider>{children}</ApolloProvider>
-    </AppToasterProvider>
+const wrapper = ({
+  children,
+  initialRoutes,
+  projectConfig,
+}: {
+  children: ReactNode
+  projectConfig?: ProjectConfig
+  initialRoutes?: string[]
+}) => (
+  <MemoryRouter initialEntries={initialRoutes}>
+    <ProjectConfigProvider projectConfig={projectConfig ?? showcaseConfig}>
+      <AppToasterProvider>
+        <ApolloProvider>{children}</ApolloProvider>
+      </AppToasterProvider>
+    </ProjectConfigProvider>
   </MemoryRouter>
 )
 
+const withCustomWrapper =
+  (initialRoutes: string[], projectConfig: ProjectConfig) =>
```

Comment:
:upside_down_face: Since we never set multiple initial routes in this file, we could simplify this to
```suggestion
  (initialRoues: string, projectConfig: ProjectConfig) =>
```

Reply:
hm the reason why i used an array was that it can be empty without an additional check.
Since passing this prop to the wrapper is optional, i then would have to check if the value is undefined before passing it to the `MemoryRouter`
You can test your suggested changes in the code